src/app/utilities/constants.ts

```diff
@@ -20,3 +20,13 @@ export enum BUTTON_STYLE {
   Tertiary = 'tertiary',
   TertiaryFull = 'tertiary-full',
 }
+
+export const HOST_URL =
+  process.env.HOST_URL || 'https://neurodiversityacademy.com';
+export const SITE_NAME = 'Neurodiversity Academy';
+export const LANGUAGES = { 'en-US': '/en-US' };
+export const LOCALE = 'en_US';
+export enum TYPE {
```

Comment:
Let's use META_TYPE. Type is an overused name, unless using it in a local scope having relevant context, better to be a bit more descriptive.